tchannel-core/pom.xml

```diff
@@ -87,6 +87,13 @@
         <dependency>
             <groupId>com.uber.jaeger</groupId>
             <artifactId>jaeger-core</artifactId>
+            <version>${com.uber.jaeger.version}</version>
```

Comment:
the version (and exclusions) is defined in the parent pom dependency management section. remove it from here.

Reply:
done